src/urls/package-url-handler.ts

```diff
@@ -223,14 +223,24 @@ export class PackageUrlHandler implements UrlHandler {
         url as string as OriginalDocumentUrl);
   }
 
-  packageRelativeToOriginalUrl(_originalPackageName: string, url: string):
+  packageRelativeToOriginalUrl(originalPackageName: string, url: string):
       OriginalDocumentUrl {
-    return url as OriginalDocumentUrl;
+    if (originalPackageName === this.bowerPackageName) {
+      return url as OriginalDocumentUrl;
+    } else {
+      return 'bower_components/' + originalPackageName + '/' + url as
```

Comment:
for component-style URls, this would start with `../`, no? Is there a reason we can treat all as app-style?

Reply:
I think my [comment above](https://github.com/Polymer/polymer-modulizer/pull/372/files#r176244380) also applies here: `PackageUrlHandler` is only used when converting in project mode, so all files that are part of dependencies will be in `bower_components/`.